### PR TITLE
Chore: updated dependencies

### LINKED ISSUE #219: No '*-dev' dependencies allowed (e.g: as-transfomer)

### PR DESCRIPTION
**Pull request by 0xSwapFeeder**
Aim's to resolve #219 from meta #205.

### CODE REVIEW

**Review by Ben-Rey — CHANGES_REQUESTED** [review]
Not sure about this. Test doesn't seem to work after update. L'et's talk about it

**Review by Ben-Rey — APPROVED** [review]

**Comment by Ben-Rey** [comment]
@0xSwapFeeder don't you think the issue was talking about packages like "@massalabs/as-types": "^0.0.3**-dev**", ?

**Comment by Ben-Rey** [comment]
> > @0xSwapFeeder don't you think the issue was talking about packages like "@massalabs/as-types": "^0.0.3**-dev**", ?
> > Not sure, could be it, but the issue name talks about dependencies, not package.json configuration for packages, or I didn't understood.

@gregLibert can you help us on this ?

**Comment by 0xSwapFeeder** [comment]
Done ! @Ben-Rey

File: packages/as-transformer/package.json [hunk 1]
```diff
@@ -13,19 +13,14 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@massalabs/as-types": "^0.0.3-dev",
+    "@massalabs/as-types": "^0.0.2",
```
Comment (Ben-Rey) [inline packages/as-transformer/package.json:16]:
```suggestion
    "@massalabs/as-types": "^1.0.0",
```

**Review by Ben-Rey — APPROVED** [review]

**Review by Ben-Rey — APPROVED** [review]

**Pull request merged**